ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v2/aemform/aemform.html

```diff
@@ -19,6 +19,26 @@
     // Editor evaluates this script and if the clientlib is not loaded by then, it will fail. Hence triggering an
     // event after clientlib load and initializing on that event.
     (function () {
+
+        const registerGuideBridgeEvents = (e) => {
+            const formContainerPath = e.detail.getPath();
+            bridge.connect(()=>{
```

Comment:
Can you add e2e test case for this complete use-case, where aem form has an overrridden thank you message and thank you page.

Reply:
Added the e2e test cases of overriding thank you message.